skbio/stats/composition.py

```diff
@@ -608,10 +610,271 @@ def centralize(mat):
 
     """
     mat = closure(mat)
-    cen = ss.gmean(mat, axis=0)
+    cen = scipy.stats.gmean(mat, axis=0)
     return perturb_inv(mat, cen)
 
 
+@experimental(as_of="0.4.0")
+def ancom(mat, cats,
+          alpha=0.05,
+          multicorr=False,
```

Comment:
What about `multiple_comparisons_correction=None`? That way this method can support other types of multiple comparisons correction in the future, and for now only `None` or `'holm-bonferroni'` is supported.